src/auto-lightbox.js

```diff
@@ -31,7 +31,7 @@ export const AutoLightboxEvents = {
  */
 export function installAutoLightboxExtension(ampdoc) {
   const {win} = ampdoc;
-  if (isAmp4Email(win.document)) {
+  if (!isAmphtml(win.document)) {
```

Comment:
Now that it's flipped, it won't work on PWA.

Reply:
Alternatives?

Reply:
Undo. :) Do we want to opt out dirty AMPs too?

Reply:
Oh, actually, `auto-lightbox` only runs on proxy origins. So if it's not tagged as `amp/⚡` (dirty amp/PWA) it wouldn't do anything anyway.

Reply:
(except wastefully loading the script)

Reply:
Oh we should not download the script if won't be used, no?

Reply:
Yeah, which is what this PR prevents 😉

Reply:
I mean we should early out for non-CDN origins here too.